drbergman/pcvct
# testing improvements and cleanup

- hpc tests
- database tests
- clean up classes and tests
- deletion tests
- fill out substrates tests
- population tests
- sensitivty cleanup and tests
- tests with ic substrates and ecm
- import the template-ecm for testing
- remove VCTClassID and add Module tests
- print git status if dirty
- better git clean check
- test dirty
- testing physicell is dirty and downloaded
- fix physicell download
- gh token for physicell download too
